docs/getting-started/customizing-your-setup.md

```diff
@@ -0,0 +1,45 @@
+---
+description: Customizing your setup
+---
+
+# Customizing your setup
+
+Once you've [added your repositories](codacy-quickstart.md), you can customize your setup to adapt the analysis tools to your use case, avoid false positives, and integrate code coverage analysis into your quality pipeline.
+
+To customize your setup, follow these steps:
+
+1.  [Ignoring files](#ignoring-files)
+1.  [Configuring tools and code patterns](#configuring-code-patterns)
+1.  [Adding coverage reports](#adding-coverage-reports) (optional)
+
+## 1. Ignoring files {: id="ignoring-files"}
+
+Ignoring non-essential files helps you avoid false positives, streamlines your quality pipeline, and encourages developers to adopt best practices and to keep following them.
```

Comment:
This part highlights not only the advantage of ignoring files but also of configuring the tools and code patterns, since if users fail to do that they'll also receive results that don't interest them and can be considered false positives.

This is why ignoring files and configuring the tools and code patterns [were grouped together under this section](https://docs.codacy.com/faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate/#adjusting-analysis-settings).

My suggestion here is that we mention these advantages directly in the intro of this page, as a way to convince users of the importance of going through the instructions (it will help make the intro more assertive).

Reply:
It probably helps to think of the page intro as the **Concept** and the sections as the **Tasks**.